projects/open-mpi.org/package.yml

```diff
@@ -0,0 +1,73 @@
+distributable:
+  url: https://download.open-mpi.org/release/open-mpi/v{{version.marketing}}/openmpi-{{version}}.tar.bz2
+  strip-components: 1
+
+versions:
+  github: open-mpi/ompi/tags
+
+dependencies:
+  open-mpi.org/hwloc: '*'
+  libevent.org: '*'
+  linux:
+    gnu.org/gcc: '*'
+
+build:
+  dependencies:
+    tea.xyz/gx/make: '*'
+    darwin:
+      gnu.org/autoconf: '*'
+      
+  script:
+    - ./configure $ARGS
+    - make --jobs {{ hw.concurrency }} all
+    - make --jobs {{ hw.concurrency }} check
+    - make --jobs {{ hw.concurrency }} install
+    - install {{prefix}}/lib/*.mod {{prefix}}/include/
+    - sed -i.bak 's|{{prefix}}|${pcfiledir}/../..|g' {{prefix}}/lib/pkgconfig/*.pc
+    
+  env:
+    darwin:
+      MACOSX_DEPLOYMENT_TARGET: "$(sw_vers -productVersion)"
```

Comment:
This should be automatically 10.6, so we support versions we intend (i.e. you shouldn't have to set it). does it not build unless set?